collections/_pages/talks-and-presentations.html

```diff
@@ -57,14 +57,15 @@
 }
 iframe{
   margin: 0 2rem 0 2rem;
-}
-.modal-window > div {
   border-radius: 1rem;
```

Comment:
I don't think we should apply curved borders to the video.

Reply:
+1

Reply:
okay will revert that, does it not look good or some design violations?

Reply:
@shubham-tyagi no not any violation, just looks a little blunt to me, you can try reducing the border-radius a bit  ( 0.5 rem maybe ), that should work too.

Reply:
yeah sure, will do that.